app/controllers/api/v1/search_controller.rb

```diff
@@ -503,9 +503,25 @@ def set_search_facets_and_filters
             facet = SearchFacet.find_by(identifier: facet_id)
             if facet.present?
               matching_filters = []
-              facet.filters.each do |filter|
-                if filter_values.include?(filter[:id])
-                  matching_filters << filter
+              if facet.is_numeric?
+                # first filter value is range, e.g. 20-40, second value is unit, e.g. years
+                min_value, max_value = self.class.split_query_param_on_delim(parameter: filter_values.first, delimiter: '-').map(&:to_f)
```

Comment:
As mentioned in our departing chat, using `-` here precludes easy range filters involving negative numbers.  

Practically speaking, all `number` fields in [the metadata convention](https://storage.googleapis.com/broad-singlecellportal-public/AMC_v1.1.3.json) are non-negative.  But negative numbers are seen e.g. with differential expression metrics like log<sub>2</sub>(fold change), and it'd be nice to not preclude that.

Let's chat tomorrow about possible solutions!